release_docs/INSTALL_CMake.txt

```diff
@@ -295,10 +295,15 @@ IV. Further considerations
             -DSZIP_LIBRARY:FILEPATH=some_location/lib/szlib.lib
             -DSZIP_INCLUDE_DIR:PATH=some_location/include
             -DSZIP_USE_EXTERNAL:BOOL=OFF
+            (For libaec use -Dlibaec_* options)
```

Comment:
Maybe (For libaec use -Dlibaec_* in place of -DSZIP_* for these options) would be clearer, if that is the actual meaning?

Reply:
That is the actual meaning. The original SZIP package (under that name) is not only carrying code with problematic licence, it is very old as a conception. Currently, the only reliable SZIP compressor and encoder is provided by Libaec. Maybe some sites still support SZIP, but I consider that to be driven by purely anthropological interest.

Reply:
This command "cmake -C ../<hdf5_src_dir>/config/cmake/cacheinit.cmake -G "Unix Makefiles" -DHDF5_ALLOW_EXTERNAL_SUPPORT:STRING="GIT" -DHDF5_ENABLE_ROS3_VFD=ON -DHDF5_ENABLE_ZLIB_SUPPORT=ON -DHDF5_ENABLE_SZIP_SUPPORT=ON -DHDF5_ENABLE_PLUGIN_SUPPORT=ON -DCMAKE_BUILD_TYPE:STRING=Release ../<hdf5_src_dir>" now downloads and builds szip from https://github.com/MathisRosenhauer/libaec.git using the code from byrnHDF:develop-libaec-fix.